app/models/comment.rb

```diff
@@ -2,4 +2,5 @@ class Comment < ApplicationRecord
   validates :body, presence: true, length: { in: 2..20 }
 
   belongs_to :posts
+  belongs_to :user_id
```

Comment:
- [ ] It shoud belong to :user (as in the User model), not user_id.

Reply:
- [ ] Also, if it belongs to one it should be :post, in the singular.

Reply:
https://guides.rubyonrails.org/association_basics.html#the-belongs-to-association